lib/fs.js

```diff
@@ -34,7 +35,7 @@ const O_WRONLY = constants.O_WRONLY || 0;
 const isWindows = process.platform === 'win32';
 
 const DEBUG = process.env.NODE_DEBUG && /fs/.test(process.env.NODE_DEBUG);
-const errnoException = util._errnoException;
```

Comment:
It would. It's probably better to add a test for this to prevent accidental breakage